nodeEditor/src/editDistance.ts

```diff
@@ -0,0 +1,261 @@
+// Derived from jsdiff by Kevin Decker et al
```

Comment:
That is a lot of code and honestly I'm not sure we want it in the Shader material editor. Why do we need to get diffs?